spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+require 'factory_bot_rails'
```

Comment:
not sure this file is required
(especially when `rails_helper.rb` already exists with `RSpec.configure do |config|`...)